### ISSUE TITLE
Ileo-cecal ulcer

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6402]
**Preferred term label:**
Ileo-cecal ulcer

**Synonyms**
Ileocecal ulcer

**Definition (free text, please give PubMed ID)**
Ulcers seen in the ileo-cecal region.
PMID: 28744345

**Parent term (use hpo.jax.org/app)**
Abnormality of the small intestine

**Comment by pnrobinson** [comment]
added
